crates/bdk/Cargo.toml

```diff
@@ -45,8 +45,9 @@ dev-getrandom-wasm = ["getrandom/js"]
 
 [dev-dependencies]
 lazy_static = "1.4"
-env_logger = "0.7"
+env_logger = { version = "0.7", default-features = false }
```

Comment:
I am a little bit picky with changes to `Cargo.toml`.
I understand the `anyhow` addition, but why changing `env_logger` to `default-features = false`?